features/donations/quick_donation.feature

```diff
@@ -9,11 +9,11 @@ Background:
   Given the following account codes exist:
   | name             | code | description                                                     | donation_prompt          |
   | Soda Fund        | 0504 | The Soda Funds aims to put a Soda Fountain in Soda Hall         |                          |
-  And I am logged in as customer "Tom Foolery"
```

Comment:
Removed Background Step to automatically login as a User as Web Step that checks if User is not logged in doesn't work as intended,
Therefore, each test in first line will specify if there's a User logged in, or no login